<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentTab</class>
 <widget class="QWidget" name="ExperimentTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="layout1">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="layout7">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QSplitter" name="host_guest_grid_splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="hosts_group_box">
        <property name="title">
         <string>Hosts</string>
        </property>
        <layout class="QVBoxLayout" name="layout6">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CustomListWidget" name="hosts_list_widget">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="batchSize">
            <number>97</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="guests_group_box">
        <property name="title">
         <string>Guests</string>
        </property>
        <layout class="QVBoxLayout" name="layout5">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CustomListWidget" name="guests_list_widget">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="grids_group_box">
        <property name="title">
         <string>SO3 grids</string>
        </property>
        <layout class="QVBoxLayout" name="layout4">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="CustomListWidget" name="grids_list_widget">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout8">
       <item>
        <widget class="QCheckBox" name="show_full_path_check_box">
         <property name="text">
          <string>Show full path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_grids_button">
         <property name="text">
          <string>Add grids</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout9" stretch="0,1">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="tabSpacing">
        <number>6</number>
       </property>
       <widget class="QWidget" name="settings_widget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>224</width>
          <height>224</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Settings</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QGroupBox" name="caging_parameters_group_box">
           <property name="title">
            <string>Caging parameters</string>
           </property>
           <layout class="QGridLayout" name="layout10">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <property name="spacing">
             <number>4</number>
            </property>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="epsilon_spin_box">
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.360000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="static1">
              <property name="text">
               <string>Epsilon:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="static2">
              <property name="text">
               <string>Infinity:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="infinity_spin_box">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="jobs_settings_group_box">
           <property name="title">
            <string>Job settings</string>
           </property>
           <layout class="QVBoxLayout" name="layout11">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QRadioButton" name="single_process_radio_button">
              <property name="text">
               <string>Run in this process</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">job_process_button_group</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="multi_process_radio_button">
              <property name="text">
               <string>Run in separate processes</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">job_process_button_group</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="Line" name="static3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="save_to_file_check_box">
              <property name="text">
               <string>Back up job results to file</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="calculation_widget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>233</width>
          <height>207</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Parameter calculation</string>
        </attribute>
        <layout class="QVBoxLayout" name="layout15">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QGroupBox" name="epsilon_calculation_group_box">
           <property name="title">
            <string>Minimal epsilon</string>
           </property>
           <layout class="QVBoxLayout" name="layout17">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="layout16" stretch="0,1">
              <item>
               <widget class="QLabel" name="static8">
                <property name="text">
                 <string>Guest:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="guests_combo_box"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layout18" stretch="0,1">
              <item>
               <widget class="QLabel" name="static9">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SO&lt;span style=&quot; vertical-align:sub;&quot;&gt;3&lt;/span&gt; resolution:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="grid_resolution_spin_box">
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
                <property name="value">
                 <double>0.054200000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layout19">
              <item>
               <widget class="QLabel" name="static10">
                <property name="text">
                 <string>Minimal epsilon:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="minimal_epsilon_spin_box">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="decimals">
                 <number>4</number>
                </property>
                <property name="maximum">
                 <double>2.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="set_minimal_epsilon_spin_box">
              <property name="text">
               <string>Set as current epsilon</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="screening_widget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>219</width>
          <height>212</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Parameters screening</string>
        </attribute>
        <layout class="QVBoxLayout" name="layout13">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QGroupBox" name="epsilon_screening_group_box">
           <property name="title">
            <string>Epsilon</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="layout14">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <property name="spacing">
             <number>4</number>
            </property>
            <item row="7" column="0" colspan="2">
             <widget class="QRadioButton" name="incremental_screening_radio_button">
              <property name="text">
               <string>Incremental screen</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">epsilon_screening_button_group</string>
              </attribute>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="minimum_epsilon_spin_box">
              <property name="maximum">
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QDoubleSpinBox" name="binary_search_precision_spin_box">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="static4">
              <property name="text">
               <string>min:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="static6">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>step:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="static5">
              <property name="text">
               <string>max:</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="static7">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>precision:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QDoubleSpinBox" name="epsilon_step_spin_box">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="maximum">
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QDoubleSpinBox" name="maximum_epsilon_spin_box">
              <property name="maximum">
               <double>2.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="10" column="0" colspan="2">
             <widget class="QRadioButton" name="binary_search_radio_button">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Search of result change</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">epsilon_screening_button_group</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="jobs_group_box">
       <property name="title">
        <string>Jobs</string>
       </property>
       <layout class="QVBoxLayout" name="layout2">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="add_jobs_button">
          <property name="text">
           <string>Add all combinations to the list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="job_splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListWidget" name="jobs_list_widget">
           <property name="font">
            <font>
             <family>Abyssinica SIL</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
          </widget>
          <widget class="QGroupBox" name="job_details_group_box">
           <property name="title">
            <string>Job details</string>
           </property>
           <layout class="QVBoxLayout" name="layout3">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextEdit" name="job_details_edit">
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>238</red>
                    <green>238</green>
                    <blue>238</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>238</red>
                    <green>238</green>
                    <blue>238</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>238</red>
                    <green>238</green>
                    <blue>238</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomListWidget</class>
   <extends>QListWidget</extends>
   <header>ui/widgets/custom_list_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="epsilon_screening_button_group"/>
  <buttongroup name="job_process_button_group"/>
 </buttongroups>
</ui>
